ykcompile/src/lib.rs

```diff
@@ -550,11 +550,12 @@ impl<TT> TraceCompiler<TT> {
                     }
                     _ => todo!(),
                 }
-                return;
+                return Location::Register(reg, false);
```

Comment:
Is the `return` keyword here necessary?

Reply:
Yes, it's an intentional early return. During development I removed it accidentally and hilarity ensued.

Reply:
Ah yes, I didn't see that it was part of the if block.